nuget/Azure.IoT.Gateway.SDK.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Azure.IoT.Gateway.SDK.Net</id>
+    <version>2017.01.13.2</version>
```

Comment:
>2017.01.13.2 [](start = 13, length = 12)

Investigate if this is supported by Nuget. I am not sure, and don't recall seeing this in packages I worked with.